packages/ember-application/tests/system/dependency_injection/to_string_test.js

```diff
@@ -1,62 +1,60 @@
-import { guidFor } from 'ember-utils';
-import { ENV, context } from 'ember-environment'; // lookup, etc
-import { run } from 'ember-metal';
-import Application from '../../../system/application';
+import { guidFor, assign } from 'ember-utils';
 import { Object as EmberObject } from 'ember-runtime';
-import DefaultResolver from '../../../system/resolver';
-
-let originalLookup, App;
-
-QUnit.module('Ember.Application Dependency Injection – toString', {
-  setup() {
-    originalLookup = context.lookup;
-
-    run(() => {
-      App = Application.create();
-      context.lookup = {
-        App: App
-      };
+import { Resolver as DefaultResolver } from 'ember-application';
+import { moduleFor, ApplicationTestCase, ModuleBasedTestResolver } from 'internal-test-helpers';
+
+moduleFor('Ember.Application Dependency Injection - DefaultResolver#toString', class extends ApplicationTestCase {
+  constructor() {
+    super();
+    this.application.Post = EmberObject.extend();
+    this.runTask(() => this.visit('/'));
+  }
+  get applicationOptions() {
+    return assign(super.applicationOptions, {
+      Resolver: DefaultResolver
```

Comment:
I added a `DefaultResolverApplicationTestCase` in a recent PR, [example here](https://github.com/emberjs/ember.js/blob/4ec05b8091a5e701ffc95f4c9f687831f7478262/packages/ember-application/tests/system/application_test.js#L189). It would allow you to avoid this setup.